### DISCUSSION TITLE
Getting quicker to edit a PR/issue description

### DISCUSSION THREAD

**Discussion by mr-robot-2008** [community/community#33943] [Repositories] [answered]
Problem
I frequent (tens of times a day) edit PRs' and issues' descriptions. I have to click twice to do it. I have not found a [keyboard shortcut](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts) to do it quicker.

Solution
Both could be applied.

A. Reduce one click
Move the "Edit" action out of the "..." icon menu to be an icon itself (e.g. a 🖊️ icon etc.). To illustrate this:

[Screenshot 2022creenshot 2022-09-01 at 15 58 31-09-01 at 15 58 31](https://user-images.githubusercontent.com/35540/187932952-8bda9db2-e337-4e26-8399-ae5924df27ae.png)

B. Add a key shortcut
Allow to click a key combination which would do the same as clicking "Edit", scroll there and put the cursor at the top of the edited text.

**Comment by Tanmay-Tiwaricyber** [comment, marked as answer]
There is "Site-wide shortcuts > Open command palette (e.g. ⌘ k or ⌘ ⌥ k)" keyboard shortcut after which you can type for example "body", hit enter and run "Edit pull request body".

Screenshot 2022-09-12 at 13 26 34

This solves my issue.
